src/appleseed/renderer/modeling/phasefunction/phasefunction.h

```diff
@@ -134,25 +135,34 @@ class APPLESEED_DLLSYMBOL PhaseFunction
         const float                 distance,                   // distance to the point on this volume segment
         Spectrum&                   spectrum) const = 0;        // resulting spectrum
 
+    // Get the scattering coefficient (spectrum) in the ray origin.
+    virtual const Spectrum& scattering_coefficient(
+        const ShadingRay&           volume_ray,                 // ray used for marching inside the volume
+        const void*                 data) const = 0;            // input values
```

Comment:
We usually pass input data as first argument. It would make sense to follow this convention for phase functions.